src/core/agent/services/credentialService.ts

```diff
@@ -184,7 +187,8 @@ class CredentialService extends AgentService {
           credential.sad.d,
           credential.sad.a.dt,
           credential.schema.title,
-          credential.sad.i
+          credential.sad.i,
+          credential.schema.id
```

Comment:
Is it not `schema.$id`? I thought this was the format it returned with `list()`

Reply:
My mistake, sorry